integration/maptest/map.go

```diff
@@ -177,55 +177,57 @@ func RunMapRevisionZero(ctx context.Context, t *testing.T, tadmin trillian.Trill
 		},
 	} {
 		for _, hsr := range tc.hashStrategy {
-			tree, hasher, smr, err := newTreeWithHasher(ctx, tadmin, tmap, hsr.hashStrategy)
-			if err != nil {
-				t.Errorf("%v: newTreeWithHasher(%v): %v", tc.desc, hsr.hashStrategy, err)
-			}
-			if got, want := smr.RootHash, hsr.wantRoot; want != nil && !bytes.Equal(got, want) {
-				t.Errorf("%v: newTreeWithHasher() returned unexpected root hash %x, want %x", tc.desc, got, want)
-			}
-			pubKey, err := der.UnmarshalPublicKey(tree.GetPublicKey().GetDer())
-			if err != nil {
-				t.Errorf("%v: UnmarshalPublicKey(%v): %v", tc.desc, hsr.hashStrategy, err)
-			}
+			t.Run(fmt.Sprintf("%v/%v", tc.desc, hsr.hashStrategy), func(t *testing.T) {
+				tree, hasher, smr, err := newTreeWithHasher(ctx, tadmin, tmap, hsr.hashStrategy)
+				if err != nil {
+					t.Fatalf("newTreeWithHasher(%v): %v", hsr.hashStrategy, err)
```

Comment:
Does t.Fatalf() kill the entire test suite or will it be recovered by test.T so the next sub-test can run?

Reply:
Fatalf only kills the test currently inside `t.Run` which is nice.